specification/mysql/resource-manager/readme.md

```diff
@@ -22,11 +22,13 @@ To see additional help and options, run:
 
 
 ### Basic Information
-These are the global settings for the Sql API.
+These are the global settings for the MySql API.
 
 ``` yaml
+title: MySQLManagementClient
+description: 'The Microsoft Azure management API provides create, read, update, and delete functionality for Azure MySQL resources including servers, databases, firewall rules, VNET rules, log files and configurations with new business model.'
 openapi-type: arm
-tag: package-2017-12-01
```

Comment:
Why remove this tag instead of just creating a new one?

Reply:
We wanted to include Private Endpoint related json in global settings. Hence reused existing tag 2018-06-01.